milli/src/search/criteria/exactness.rs

```diff
@@ -227,14 +227,14 @@ fn resolve_state(
                     // compute intersection on pair of words with a proximity of 0.
                     Phrase(phrase) => {
                         let mut bitmaps = Vec::with_capacity(phrase.len().saturating_sub(1));
-                        for words in phrase.windows(2) {
-                            if let [left, right] = words {
```

Comment:
I'm ignoring stop words in Exact too. I'm not sure if it was the right thing to do as the behaviour is now changed. Is this correct?

Reply:
Done in bc938bf3795976e26a748c05787f5112934d717e